README.md

```diff
@@ -201,22 +201,22 @@ domJSON is a global variable to store two methods: `.toJSON()` to convert a DOM
 #### domJSON.toJSON(node, [opts]) ⇒ <code>Object</code> \| <code>string</code>
 Take a DOM node and convert it to simple object literal (or JSON string) with no circular references and no functions or events
 
-| Param | Type | Description |
```

Comment:
I have my code editor to prettify on save - hope these changes don't bother.

<img width="1343" alt="screen shot 2018-08-12 at 11 05 15 am" src="https://user-images.githubusercontent.com/792606/44001339-a31e5dd4-9e1f-11e8-951f-e0508ae86d9b.png">